iceoryx_hoofs/test/moduletests/test_access_control.cpp

```diff
@@ -251,9 +248,9 @@ TEST_F(AccessController_test, addNameInWrongPlace)
 TEST_F(AccessController_test, addManyPermissions)
 {
     ::testing::Test::RecordProperty("TEST_ID", "998c828b-8b9e-4677-9c36-4a1251c11241");
-    AccessController::string_t groupName = "root";
+    AccessController::permissionString_t groupName = "root";
 
-    bool entryAdded;
+    bool entryAdded = false;
```

Comment:
AUTOSAR prefers 
```suggestion
    bool entryAdded{false};
```
If we change it on this PR, we won't have to care about it anymore. Same for the other variable above.